runtime/services/catalog/artifacts/yaml/objects.go

```diff
@@ -425,6 +446,21 @@ func fromMetricsViewArtifact(metrics *MetricsView, path string) (*drivers.Catalo
 	}, nil
 }
 
+func validatedPolicyFieldList(fieldConditions []*ConditionalColumn, dimensions map[string]bool, property string) error {
+	if len(fieldConditions) > 0 {
+		for _, field := range fieldConditions {
+			if field.Name == "" || field.Condition == "" {
+				return fmt.Errorf("invalid 'policy': '%q' fields must have a valid name and condition", property)
```

Comment:
1. In the YAML, the fields are "name" and "if" (not "condition")
2. It should put single quotes around `name` and `condition` since they're YAML fields (like for `'policy'`)